Sources/Plasma/CoreLib/HeadSpin.h

```diff
@@ -286,10 +286,6 @@ template <> inline double hsToLE(double value) { return hsToLEDouble(value); }
 #   ifndef fileno
 #       define fileno(__F) _fileno(__F)
 #   endif
```

Comment:
btw, I am tempted to remove this one as well, in the interest of getting random rarely-used platform handling out of the core header that's included everywhere. But this macro is actually used in two places and thus not completely pointless.